scarb/tests/local_registry.rs

```diff
@@ -123,7 +123,9 @@ fn url_pointing_to_file() {
     drop(registry_t);
 }
 
+// FIXME(#883)
```

Comment:
```suggestion
// FIXME(#883): Unignore these tests.
```

otherwise editors may not catch this

Reply:
Thank you!